src/exported-tests/component-tests.ts

```diff
@@ -0,0 +1,39 @@
+import merge from "lodash-es/merge";
```

Comment:
We knew you weren't going to have this file via PAL at this stage. I'm curious if there was anything (other than formatting for TypeScript) that was needed in order to meet your needs.  I'm excited to get more of these files in a shared location for everyone to easily use.

Reply:
I don't think there aren't any other features we would need (besides the typescript formatting) from the base Component Tests file. Once that is available in a shared location we will definitely switch over to utilizing that 👍

Reply:
Thank you so much for reviewing!!